.github/workflows/ci.yaml

```diff
@@ -12,16 +12,18 @@ name: CI
 on:
   workflow_dispatch:
     inputs: {}
-  push:
-    branches:
-      - master
+  # disabled as we have ci-multiarch.yaml now
```

Comment:
Well, it not a good motivation to disable single-arch since we used to have multiarch build only, but it was not stable.

Now the situation is better and multiarch build is reworked from buildx:platforms which we used to use https://github.com/eclipse/che-dashboard/pull/166/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddL57

But there is a risk that multiarch build(more often s390x) will fail without any guilt from Dashboard repo side.
On Che Community I raised a topic to check if somebody is interested in multiarch build, and no-one complained, so it's why we decided to keep next as the default and CI-Multiarch is optional.

I don't see a need in amd64-next, if we don't need it, then let's remove.

Reply:
ok, deleted the file .github/workflows/ci.yaml